src/js/components/Popover.jsx

```diff
@@ -2,22 +2,23 @@
 
 import React, { Component, PropTypes } from 'react';
 import FloatingElement, { positions, positionValues, sanitizePosition } from './FloatingElement';
+import {ModalContainer} from '.';
 
 export class Popover extends Component {
 
     //--------------------------------------
     //  User Interaction
     //--------------------------------------
 
-    modalScreenClicked:HTMLEventHandler = () => {
+    modalScreenClicked = () => {
```

Comment:
I think it's a wash in readability, but it's mostly so that the value is the same, otherwise every time render() is called you'll get a different value (!==) for the closure even if there's no locals bound, and so it's another thing react has to change in the DOM. I'd prefer to use a `@bind` annotation on a normal method, but that requires adding a bit more to our babel setup.